config/task.config

```diff
@@ -0,0 +1,54 @@
+process {
+
+    withName: Cutadapt_1 {
+        ext.prefix = {"${meta.id}_1"}
+        container='quay.io/biocontainers/cutadapt:4.6--py39hf95cd2a_1'
+        ext.args ={" -a $params.linked_adapter_1 --discard-untrimmed "}
+    }
+
+     withName: Cutadapt_2 {
+        ext.prefix =  {"${meta.id}_2"}
+        container='quay.io/biocontainers/cutadapt:4.6--py39hf95cd2a_1'
+        ext.args ={" -a $params.linked_adapter_2 --discard-untrimmed "}
+    }
+
+    withName: Cutadapt {
+        publishDir = { "${params.outdir}/${meta.id}/1:Trimmed_reads/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
```

Comment:
`{task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()` is not very clear and you use this multiple times. Could you turn this into a function of sorts?

```
my_human_readable_description_of_string_operation(task):
    return task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()
```
Or however groovy syntax works.